StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/LinkPaymentController.swift

```diff
@@ -0,0 +1,221 @@
+//
+//  LinkPaymentController.swift
+//  StripePaymentSheet
+//
+//  Created by Bill Meltsner on 12/9/22.
+//  Copyright © 2022 Stripe, Inc. All rights reserved.
+//
+
+import UIKit
+@_spi(STP) import StripePayments
+
+
+/// `LinkPaymentController` encapsulates the Link payment flow, allowing you to let your customers pay with their Link account.
+/// You probably want to [use the Mobile Payment Element instead](https://stripe.com/docs/payments/accept-a-payment?platform=ios&ui=payment-sheet), which supports Link alongside credit cards, bank transfers, and many other payment methods to offer your customers more choice and convenience in how they pay.
+@available(iOSApplicationExtension, unavailable)
+@available(macCatalystApplicationExtension, unavailable)
+@_spi(LinkOnly) public class LinkPaymentController {
```

Comment:
Yeah... it's a bit complicated, as the status is less "private beta" and more "there's like one or two merchants we expect to actually use this" - but I'll check with the project leads and see if they're comfortable with that callout, could be a way to assess merchant interest.

Reply:
I think generally we need to explain who can/should use this and set expectations here, since it's not GA e.g. by saying it's in "Private alpha/beta". Or if it is GA, remove the `@_spi(LinkOnly)` attribute.

Reply:
Updated this comment based on discussion with the PM on this project.